int32[] face_labels
geometry_msgs/Point[] centroids
bool[] mutualGaze